tests/yam/agama/agama.pm

```diff
@@ -18,14 +20,25 @@ use testapi qw(
 sub run {
     my $self = shift;
     my $test = get_required_var('AGAMA_TEST');
+    my $configure_dasd = get_var('CONFIGURE_DASD');
     my $reboot_page = $testapi::distri->get_reboot_page();
 
+    # For testing purposes remove before merge
+    if (is_s390x() && is_backend_s390x() && get_var('AGAMA_DELAY')) {
+      print("Sleeping 30 minutes for testing purposes");
```

Comment:
you don't have your PR in WIP or DRAFT, code like this should not exists unless you PR signal that, it is dangerous in the sense that could be merged accidentally.

Reply:
Fixed 4740dff6fd12210421ee398a025c7823e0e963f7